libs/cli/DOCS.md

```diff
@@ -32,182 +32,188 @@ $ langchain hub [OPTIONS] COMMAND [ARGS]...
 
 **Commands**:
 
-* `new`: Creates a new hub package.
-* `start`: Starts a demo LangServe instance for this...
+* `add`: Adds the specified package to the current...
+* `install`
+* `list`: Lists all packages in the current...
+* `new`: Create a new LangServe application.
+* `remove`: Removes the specified package from the...
+* `serve`: Starts the LangServe instance.
+
+### `langchain app add`
 
-### `langchain hub new`
+Adds the specified package to the current LangServe instance.
 
-Creates a new hub package.
+e.g.:
+langchain app add extraction-openai-functions
+langchain app add git+ssh://git@github.com/efriis/simple-pirate.git
 
 **Usage**:
 
 ```console
-$ langchain hub new [OPTIONS] NAME
+$ langchain app add [OPTIONS] [DEPENDENCIES]...
 ```
 
 **Arguments**:
 
-* `NAME`: [required]
+* `[DEPENDENCIES]...`: The dependency to add
 
 **Options**:
 
+* `--api-path TEXT`: API paths to add
```

Comment:
im not sure at all what this does?

Reply:
This is the directory it gets installed in under packages/

Options:
- destination_path
- path
- save_as
- ...